BUILDING.md

```diff
@@ -526,7 +526,7 @@ to run it again before invoking `make -j4`.
 
 ##### Option 1: Manual install
 
-* [Python 2.7](https://www.python.org/downloads/)
+* [Python 3.8](https://www.python.org/downloads/)
```

Comment:
Why not https://www.microsoft.com/en-us/p/python-38/9mssztt1n39l ?

Reply:
I don't know if it's available on Windows 8

Reply:
I think no. The System Requirements on MS store said only for Windows10